src/rgw/rgw_rest_s3.cc

```diff
@@ -3991,6 +3991,8 @@ int RGWHandler_REST_S3::init(RGWRados *store, struct req_state *s,
   const char *sc = s->info.env->get("HTTP_X_AMZ_STORAGE_CLASS");
   if (sc) {
     s->info.storage_class = sc;
+  } else {
```

Comment:
if that's the case, could just put
```c++
  s->info.storage_class = s->info.env->get("HTTP_X_AMZ_STORAGE_CLASS",
					   RGW_STORAGE_CLASS_STANDARD);

```

Reply:
> if that's the case, could just put
> 
> ```c++
>   s->info.storage_class = s->info.env->get("HTTP_X_AMZ_STORAGE_CLASS",
> 					   RGW_STORAGE_CLASS_STANDARD);
> ```

Yes, your suggestion is better.
Thanks.